velox/exec/fuzzer/DuckQueryRunner.cpp

```diff
@@ -448,6 +500,17 @@ std::optional<std::string> DuckQueryRunner::toSql(
 
 std::optional<std::string> DuckQueryRunner::toSql(
     const std::shared_ptr<const core::NestedLoopJoinNode>& joinNode) {
+  std::string probeTableName;
```

Comment:
nit: maybe move this part out as a function as its being used in a couple of places...

Reply:
Done.